datasets/goes/goes-glm/collection/template.json

```diff
@@ -0,0 +1,334 @@
+        }
+    ],
+    "stac_extensions": [
+        "https://stac-extensions.github.io/processing/v1.1.0/schema.json",
+        "https://stac-extensions.github.io/scientific/v1.0.0/schema.json",
+        "https://stac-extensions.github.io/table/v1.2.0/schema.json",
+        "https://stac-extensions.github.io/item-assets/v1.0.0/schema.json"
+    ],
+    "keywords": [
+        "GOES",
+        "NOAA",
+        "NASA",
+        "Satellite",
+        "Lightning",
+        "Weather"
+    ],
+    "msft:short_description": "Continuous lightning detection over the Western Hemisphere from the Geostationary Lightning Mapper (GLM) instrument.",
+    "msft:storage_account": "goeseuwest",
+    "msft:container": "noaa-goes-geoparquet",
+    "msft:group_id": "goes",
+    "providers": [
+        {
+            "name": "DOC/NOAA/NESDIS",
+            "description": "Provided by:\n\n* U.S. Department of Commerce\n* National Oceanic and Atmospheric Administration\n* National Environmental Satellite, Data, and Information Services",
+            "roles": [
+                "producer",
+                "licensor"
+            ],
+            "url": "https://www.goes.noaa.gov"
+        },
+        {
+            "name": "Microsoft",
+            "roles": [
+                "processor",
+                "host"
+            ],
+            "url": "https://planetarycomputer.microsoft.com"
+        }
+    ],
+    "assets": {
+        "thumbnail": {
+            "title": "GOES GLM Thumbnail",
+            "href": "https://ai4edatasetspublicassets.blob.core.windows.net/assets/pc_thumbnails/goes-glm-thumb.png",
+            "type": "image/png",
+            "roles": [
+                "thumbnail"
+            ]
+        },
+        "geoparquet-items": {
+            "href": "abfs://items/goes-glm.parquet",
+            "type": "application/x-parquet",
+            "roles": [
+                "stac-items"
+            ],
+            "title": "GeoParquet STAC items",
+            "description": "Snapshot of the collection's STAC items exported to GeoParquet format.",
+            "msft:partition_info": {
+                "is_partitioned": true,
+                "partition_frequency": "QS"
+            },
+            "table:storage_options": {
+                "account_name": "pcstacitems"
+            }
+        }
+    },
+    "extent": {
+        "spatial": {
+            "bbox": [
+                [
+                    -180.0,
+                    -66.56,
+                    -8.44,
+                    66.56
+                ],
+                [
+                    156.44,
+                    -66.56,
+                    180.0,
+                    66.56
+                ]
+            ]
+        },
+        "temporal": {
+            "interval": [
+                [
+                    "2018-02-13T16:10:00Z",
+                    null
+                ]
+            ]
+        }
+    },
+    "item_assets": {
+        "geoparquet_events": {
```

Comment:
Is there a reason you're using `geoparquet-{name}` instead of just `{name}` for these asset keys? I have a slight preference for calling these asset keys `events`, `flashes`, and groups`.

I see that the `netcdf` file is being cataloged as well, which is great. But since we're pushing users to the geoparquet assets I would say give them the "default" names without any file format qualifier.

Reply:
I'm good with that. I'll fix this in pctasks, as these types of changes tend to produce disagreements that are not worth the time given the subjective/personal-preference nature. But, yeah, I think your reasoning is sound.

Reply:
My preference would be to match stactools-goes above all else. I'll make an issue there.

Reply:
Done

Reply:
>My preference would be to match stactools-goes above all else. I'll make an issue there.

I see your issue. Thanks.